get_lc_instance_type.py

```diff
@@ -0,0 +1,34 @@
+from utils import http_error_handling
+import boto3
+from fleece.xray import monkey_patch_botocore_for_xray
+import os
+
+monkey_patch_botocore_for_xray()
+
+cache = dict()
+
+
+def get_lc_instance_type(client, lc_name):
+    desc = client.describe_launch_configurations(
+        LaunchConfigurationNames=[lc_name]
+    )
+    if not len(desc['LaunchConfigurations']):
+        raise Exception('LC %s not found' % lc_name)
+    lc = desc['LaunchConfigurations'][0]
+    used_instance_type = lc['InstanceType']
+    print("%s uses instance type %s" % (lc_name, used_instance_type))
```

Comment:
If you're using python3.6 it is recommended to use the new string formatting syntax: f'{lc_name} uses instance type {used_instance_type}'